src/components/Nav.js

```diff
@@ -151,66 +151,48 @@ class Nav extends React.Component {
+                            </button>
+                        </div>
                     </div>
-                    { shouldRenderLogo ?        
-                        <Link to="/" className="logo-container">
-                            <img className="logo" src={TheiaLogoDark} alt="theia logo" />
-                        </Link>: null
-                    }
-                    {
-                        this.state.isNavRendered &&
-                        <ul className="nav__items">
-                            <li className="nav__item" onClick={event}>
-                                <Link to="/#features" className="nav__link">Features</Link>
-                            </li>
-                            <li className="nav__item" onClick={event}>
-                                <Link to="/docs/" className="nav__link" activeClassName="active">Documentation</Link>
-                            </li>
-                            <li className="nav__item" onClick={event}>
-                                <a href="https://spectrum.chat/theia" target="_blank" rel="noopener noreferrer" className="nav__link">Community</a>
-                            </li>
-                            <li className="nav__item" onClick={event}>
-                                <a href="https://typefox.io/eclipse-theia" className="nav__link" target="_blank" rel="noopener">Support</a>
-                            </li>
-                            <li className="nav__item" onClick={event}>
-                                <a href="https://typefox.io/trainings-2" className="nav__link" target="_blank" rel="noopener">Training</a>
-                            </li>
-                        </ul>
-                    }
+                    <ul className={`nav__items ${this.state.isNavRendered ? 'navIsRendered' : 'navIsNotRendered' }`}>
+                        <li className="nav__item">
+                            <Link to="/#features" className="nav__link">Features</Link>
+                        </li>
+                        <li className="nav__item">
+                            <Link to="/docs/" className="nav__link" activeClassName="active">Documentation</Link>
+                        </li>
+                        <li className="nav__item">
+                            <a href="https://spectrum.chat/theia" target="_blank" rel="noopener noreferrer" className="nav__link">Community</a>
+                        </li>
+                        <li className="nav__item">
+                            <a href="https://typefox.io/eclipse-theia" className="nav__link" target="_blank" rel="noopener">Support</a>
+                        </li>
+                        <li className="nav__item">
+                            <a href="https://typefox.io/trainings-2" className="nav__link" target="_blank" rel="noopener">Training</a>
```

Comment:
While we're at it, this link is no longer current:

```suggestion
                            <a href="https://www.typefox.io/trainings/" className="nav__link" target="_blank" rel="noopener">Training</a>
```